subsys/mpsl/fem/nrf21540_gpio/mpsl_fem_nrf21540_gpio.c

```diff
@@ -4,26 +4,41 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
+#if defined(CONFIG_MPSL_FEM_NRF21540_GPIO)
+
 #include <zephyr/devicetree.h>
 #include <zephyr/drivers/gpio.h>
 #include <string.h>
 #include <zephyr/sys/__assert.h>
+
+#if !defined(CONFIG_MPSL_FEM_PIN_FORWARDER)
 #include <mpsl_fem_config_nrf21540_gpio.h>
-#include <mpsl_fem_config_simple_gpio.h>
 #include <nrfx_gpiote.h>
 #if IS_ENABLED(CONFIG_HAS_HW_NRF_PPI)
 #include <nrfx_ppi.h>
 #elif IS_ENABLED(CONFIG_HAS_HW_NRF_DPPIC)
 #include <nrfx_dppi.h>
 #endif
+#else /* !defined(CONFIG_MPSL_FEM_PIN_FORWARDER) */
+#include <soc_secure.h>
+#if IS_ENABLED(CONFIG_PINCTRL)
+#include <pinctrl_soc.h>
+#endif
+#endif /* !defined(CONFIG_MPSL_FEM_PIN_FORWARDER) */
+
+#if !DT_NODE_EXISTS(DT_NODELABEL(nrf_radio_fem))
```

Comment:
Probably not mandatory as this file wouldn't even be compiled if nrf_radio_fem node is not present

Reply:
Removed these unnecessary `#if` blocks.